data/en/1_0005.json

```diff
@@ -47,6 +47,8 @@
                 "primary_content": [{
                     "id": "get-started",
                     "content": [{
+                        "description": "<div class=\"panel panel--simple panel--info\"><div class=\"panel__body\"><h2 class=\"venus\">Important notice for December period</h2><p>We understand that many UK businesses have ‘holiday shutdowns’ in December. If this applies to your business, please share your weekly wage figures from a more representative week.</p></div></div>"
+                    }, {
```

Comment:
should we use unicode for quotes?

Reply:
Yes. There is a card in the backlog to remove the use of escape sequences in survey JSON files.

Reply:
corrected